fastlane_bot/helpers/txhelpers.py

```diff
@@ -132,9 +133,9 @@ def validate_and_submit_transaction(
             self.cfg.logger.info(f"Waiting for transaction {tx_hash} receipt")
             tx_receipt = self._wait_for_transaction_receipt(tx_hash)
             self.cfg.logger.info(f"Transaction receipt: {dumps(tx_receipt, indent=4)}")
-            return tx_hash, tx_receipt
+            return tx_hash, tx_receipt, log_dict
```

Comment:
Passing an additional input parameter just in order to return it seems futile.
You can easily determine on the caller function that `log_dict` is valid if and only if `tx_hash` and `tx_receipt` are valid.